src/adapter.js

```diff
@@ -159,7 +158,10 @@ class MongooseAdapter {
   async getTransaction () {
     if (this.isSynced) {
       const session = await this.getSession()
-      if (!session._serverSession.isTxnActive()) await session.startTransaction()
+      if (!session._serverSession.isTxnActive()) {
+        await session.startTransaction()
+        console.warn('Transaction started. To commit changes use adapter.commitTransaction() or to abort use adapter.abortTransaction()')
```

Comment:
Can we make a warning only if user forgot to commit\abort transaction?

It can be understood wrong, if you will throw a warning when everything is ok. I mean, imagine the case. You open the transaction, do changes and commit it - everything is ok. But instead, you, as a user, get warning that you should not forgot about commit. It can be interpreted wrong.

Maybe somehow move the warning like "You are trying to do something, but we see that you have not closed current transaction" or something like that.

Reply:
Hmm, very true. Currently the only logical place for that would be under `close` function, but some users do not ever close the connection by themselves, so they might never commit changes and are left to wonder why.

But maybe that could be emphasized through documentation and examples.